translations/translations-workflow.md

```diff
@@ -24,72 +23,46 @@ as slug), with many *components*:
-   created on the corresponding GitHub repository (called *upstream* by the
-   weblate documentation).
-
-As it is managed with `git`, and as we use *bilingual* files (each translation
-files has two languages), which supposes to update the catalogs, conflicts can
-happen. To avoid such conflicts, [the documentation][2] proposes something like
-the following:
-
-*You'll need to [install and configure the `wlc` client][4]*.
-
-1. The `dev` branch (*upstream*) is moving. Developers do not
-   `extract_messages` nor `update_catalog`.
-1. From time to time, the person responsible for the translations updates
-   weblate:
-    1. Ensure weblate is in a clean state: all changes are commited (`wlc
-       commit rero_plus/<component>`).
-    1. Block the translations on the component (`wlc lock
-       rero_plus/<component>`).
-    1. Push weblate on `rero:dev` (`wlc push rero_plus/<component>`).
-    1. Pull the weblate commit locally (`git pull`).
-    1. Extract the messages and update the catalogs. Commit the changes.
-    1. Push *upstream* (`git push`).
-    1. Verify that weblate is being automatically updated.
-    1. Unlock translations on weblate (`wlc unlock rero_plus/<component>`).
-1. Translators can work on weblate.
-1. Weblate creates PR on the `dev` branch of the GitHub repository. 
-1. Weblate PR are being merged into `dev`.
-
-The section 2 could be scripted, once it's validated.
+**We do not use the automatic workflow through the GitHub webhook, to avoid
+conflicts.**
+
+The starting situation is just after a release: the GitHub `translations` branch
+is on the same commit than the [weblate]][12] `translations` branch. Also, the
+GitHub `translations` branch is on the same commit than the GitHub `dev`
+branch.
+
+1. Some changes are done on the `dev` branch.
+1. Some changes are done on weblate.
+1. Ensure that on weblate, all changes is committed.
+1. Lock the component.
+1. Then ensure that on GitHub all translations pull requests are merged into
+   `translations` branch.
+1. Fetch and pull new upstream commits (`translations` branch) on your local
+   `translations` branch.
+1. Maybe, you want to squash several commits into one.
+1. Fetch new upstream commits (`dev` branch).
+1. Rebase your `translations` branch on the upstream `dev` branch.
+1. Extract messages, check that everything looks good.
+1. Update catalogs.
+1. Commit changes in the precedent translation commit.
+1. Force push the `translations` branch upstream (`translations` branch too).
+1. On weblate, through the repository management settings, reset the weblate
+   branch on the GitHub `translations` branch (The red *maintenance* dropdown
+   menu).
+1. Unlock the component.
+1. Inform on the translations Gitter room that the component is updated and
+   open to translation.
+1. Repeat once a week, or when several US are merged into `dev` during the
+   sprint.
+1. Before release process, and even a few day before, when every weblate
```

Comment:
Instead of merging, I create a PR on `dev`, but no matter..

Reply:
I'm not sure to understand. Of course, to integrate changes into `dev`, it goes through a PR. But at the end, the proposed changes are merged into `dev`.